packages/graph-explorer/src/modules/SearchSidebar/useEntityAttributesAsScalars.tsx

```diff
@@ -0,0 +1,20 @@
+import { createResultScalar } from "@/connector/entities";
+import { type DisplayAttribute, useQueryEngine } from "@/core";
+
+/**
+ * Converts the attributes to a list of scalars. If there are no attributes, a
+ * scalar with the label "No attributes" is returned.
+ *
+ * @param attributes The attributes to convert
+ */
+export function useEntityAttributesAsScalars(attributes: DisplayAttribute[]) {
+  const queryEngine = useQueryEngine();
+  const noAttributesLabel =
+    queryEngine === "sparql" ? "No predicates" : "No attributes";
```

Comment:
Consider making the "No predicates" & "No attributes" labels a constant.  Ideally, this is a top-level UI element that can be whitelisted for different databases.

Reply:
Also, why no "No properties" instead of "No attributes"?  I thought these were generally understood to be "properties" on the graph?  Maybe I'm thinking too much about Neptune.